test/e2e/autoscale_test.go

```diff
@@ -51,7 +51,7 @@ func isDeploymentScaledToZero() func(d *v1beta1.Deployment) (bool, error) {
 	}
 }
 
-func generateTrafficBurst(clients *test.Clients, logger *zap.SugaredLogger, want int, domain string) error {
+func generateTrafficBurst(clients *test.Clients, logger *logging.BaseLogger, num int, domain string) {
```

Comment:
You dropped the returned `error` type here.

Reply:
IT was dropped by another checkin. Synced forward.